client/src/app/routes/stardust/OutputsList.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { IOutputResponse } from "@iota/iota.js-stardust";
+import React, { useEffect, useState } from "react";
+import { RouteComponentProps, useLocation } from "react-router-dom";
+import { ServiceFactory } from "../../../factories/serviceFactory";
+import { STARDUST } from "../../../models/config/protocolVersion";
+import { StardustTangleCacheService } from "../../../services/stardust/stardustTangleCacheService";
+import Pagination from "../../components/Pagination";
+import Spinner from "../../components/Spinner";
+import Output from "../../components/stardust/Output";
+import "./OutputsList.scss";
+import OutputsListProps from "./OutputsListProps";
+
+interface OutputsListState {
+    outputIds: string[];
+    tag: string;
+}
+
+interface OutputsListResponse {
```

Comment:
This would be better as `OutputListItem`, cause that's what it is.

Reply:
Done.